src/Application/Commands/HandleOutOfSyncFunds.php

```diff
@@ -74,8 +77,17 @@ protected function doExecute(InputInterface $input, OutputInterface $output): in
         $numFundingsCorrect = 0;
         $numFundingsOvermatched = 0;
         $numFundingsUndermatched = 0;
+
+        $problemIds = [];
         /** @var CampaignFunding[] $fundings */
-        $fundings = $this->campaignFundingRepository->findAll();
+        if ($mode === '2024-06-05-patch') {
+            $problemIds = [29822];
```

Comment:
doesn't need to be changed now but $problemIds is always the same when it's used, it could be moved outside of conditionals or even made a class constant, to avoid having to set it to empty just for the sake of Psalm. Could also be renamed to something like `$mat369_problemFundingIds`. I don't think it's worth actually making either of these changes now since this is just temporary code, just noting here to get shared understanding of what we're doing.

Reply:
Yeah I was just going for the shortest path from the last time we did a somewhat similar change – since as you say we will hopefully delete it again by tomorrow.